deploy/charts/rook-ceph/templates/deployment.yaml

```diff
@@ -125,3 +126,10 @@ spec:
         emptyDir: {}
       - name: webhook-cert
         emptyDir: {}
+      securityContext:
+        runAsUser: 1000
+        runAsGroup: 3000
```

Comment:
Where is the user 1000, group 3000, and fsGroup 2000 defined?

Reply:
I think I answered this in my other comment, but this is just convention.

https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-pod

The security context will force all binaries to run as the specified user and group regardless of what the container wants.